WalletWasabi.Fluent/Behaviors/WindowSizeBehavior.cs

```diff
@@ -15,8 +16,6 @@ protected override void OnAttached(CompositeDisposable disposables)
 			.Take(1)
 			.Subscribe(_ =>
 			{
-				SetWindowSize(AssociatedObject);
-
 				AssociatedObject
```

Comment:
Can you modify the behavior to have only this:
```c#
AssociatedObject
	.WhenAnyValue(x => x.Bounds)
	.Skip(1)
	.Where(b => !b.IsEmpty && AssociatedObject.WindowState == WindowState.Normal)
	.Subscribe(b =>
	{
		Services.UiConfig.WindowWidth = b.Width;
		Services.UiConfig.WindowHeight = b.Height;
	})
	.DisposeWith(disposables);
```

Or it can be removed completely and move the logic above to `ApplicationStateManager`. Both are ok to me.

Reply:
I guess it can be moved to ApplicationStateManager, the logic is only for updating the config as is now.

Reply:
ACK